migration/idempotent/007-tracing-tags.sql

```diff
@@ -96,10 +96,57 @@ $fnc$
 $fnc$;
 GRANT EXECUTE ON FUNCTION _ps_trace.tag_v_eq_matching_tags(_tag_key pg_catalog.text, _value pg_catalog.jsonb) TO prom_reader;
 COMMENT ON FUNCTION _ps_trace.tag_v_eq_matching_tags
-IS 'This function is a part of custom _ps_trace.tag_v type which is a wrapper for 
+IS 'This function is a part of custom _ps_trace.tag_v type which is a wrapper for
 the built-in jsonb. The tag_map_rewrite support function, attached to tag_v_eq,
 will use this function instead, if it can.';
 
+CREATE OR REPLACE FUNCTION _ps_trace.text_matches(_value pg_catalog.text)
+    RETURNS pg_catalog.jsonb[]
+    LANGUAGE sql IMMUTABLE
```

Comment:
NOTE: This is potentially too loose, because we end up calling `to_jsonb` which is marked `STABLE`.
I believe it has to do with the fact that `to_jsonb` deals with `ANYELEMENT` and thus its result can depend on GUCs. This is probably of no concern to us.

Reply:
added a comment